doc/src/manual/packages.md

```diff
@@ -680,7 +680,7 @@ and do not want to support the current stable version of Julia. If you choose to
 add `julia 0.y-` to your `REQUIRE`. Just remember to change the `julia 0.y-` to `julia 0.y` in
 your `REQUIRE` file once `0.y` is officially released. If you don't edit the dash cruft you are
 suggesting that you support both the development and stable versions of the same version number!
-That would be madness.  See the [Requirements Specification](@ref) for the full format of `REQUIRE`.
+That would be madness. See the [Requirements Specification](@ref) for the full format of `REQUIRE`.
 
 Lastly, in many cases you may need extra packages for testing. Additional packages which
 are only required for tests should be specified in the `test/REQUIRE` file. This `REQUIRE`
```

Comment:
Which example? JuliaParser is for parsing Julia code so it's a bit of an exception, the usual thing point 2 is about is to avoid putting Julia in the name if it's just for doing some other task and happens to be written in Julia.